manifests/init.pp

```diff
@@ -35,6 +39,10 @@
   String $map_source,
   Optional[String] $service = undef,
   Optional[String] $service_provider = undef,
+  String $username = 'tftp',
```

Comment:
I think this should be root by default and tftp on Debian. It should also be passed into the xinetd resource (which accepts a user parameter).